thoth/package_analyzer/python/fetch.py

```diff
@@ -38,11 +38,22 @@ def fetch(self, package_name: str, package_version: str) -> dict:
             "Fetching digests for package %r in version %r from %r", package_name, package_version, self.source.url
         )
 
-        artifacts = self.source.get_package_hashes(package_name, package_version, True)
+        error = False
+        error_message = ""
+        artifacts = []
+
+        try:
+            artifacts = self.source.get_package_hashes(package_name, package_version, True)
+        except Exception as e:
```

Comment:
@fridex Do you think this is the right way of capturing the error? Or should I create a new exception in `exceptions`?

Reply:
Having specific exceptions is always a good idea, syn?